scripts/ImportGeoDB.java

```diff
@@ -0,0 +1,46 @@
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.model.CityResponse;
+import com.maxmind.geoip2.record.City;
+import com.maxmind.geoip2.record.Country;
+import com.maxmind.geoip2.record.Location;
+import com.maxmind.geoip2.record.Postal;
+import com.maxmind.geoip2.record.Subdivision;
+import java.io.File;
+import java.net.InetAddress;
+
+public class ImportGeoDB {
+
+  public static void main(String[] args) throws Exception{
+    // A File object pointing to your GeoIP2 or GeoLite2 database
+    File database = new File("./data/GeoLite2-City.mmdb");
+
+// This creates the DatabaseReader object. To improve performance, reuse
+// the object across lookups. The object is thread-safe.
+    DatabaseReader reader = new DatabaseReader.Builder(database).build();
+
+    InetAddress ipAddress = InetAddress.getByName("128.101.101.101");
```

Comment:
Not knowing anything about structure of the InetAddress class I should ask:

Do they have something similar to inet_aton()? Typically getByName means lookup in DNS. 

Check:

man inet_aton
man getnostbyname

Reply:
No, InetAddress is probably the most common structure. 
Of course it's possible to represent IP in a 32-bit string but you'd have to convert back and forth...